did/document.go

```diff
@@ -280,7 +280,12 @@ func NewVerificationMethod(id DID, keyType ssi.KeyType, controller DID, key cryp
 		if err != nil {
 			return nil, err
 		}
-		vm.PublicKeyJwk = jwkAsMap
+		// go to JSON and back to fix encoding of key material
```

Comment:
can we mention why we need this and `jwkAsMap` isn't sufficient?

Reply:
so it removes lib specific types.